frontend/src/main/utils/sectionUtils.js

```diff
@@ -104,3 +105,20 @@ export const renderInfoLink = ({ cell: { value } }) => (
     </a>
   </p>
 );
+
+export function onDeleteSuccess(message) {
+  console.log(message);
+  toast(message);
+}
+
+export function cellToAxiosParamsDelete({ cell, psId }) {
+  const enrollCode = cell.row.values["classSections[0].enrollCode"];
+  return {
+    url: "/api/personalSections/delete",
```

Comment:
```suggestion
    url: "/api/courses/user",
```

Why not use the existing endpoint?